src/lib/kw_db.sh

```diff
@@ -275,6 +275,14 @@ function select_from_where()
   cmd_manager "$flag" "$cmd"
```

Comment:
As each commit should be atomic, the "code that tests the code" (tests) and documentation should be included in the same commit. So, you should squash this commit with the other one.